README.md

```diff
@@ -32,14 +32,14 @@ docker run --rm -d \
 	-e icingaweb.resources.icingaweb_db.dbname=icingaweb \
 	-e icingaweb.resources.icingaweb_db.username=icingaweb \
 	-e icingaweb.resources.icingaweb_db.password=123456 \
-	-e icingaweb.resources.icingaweb_db.charset=utf8 \
+	-e icingaweb.resources.icingaweb_db.charset=utf8mb4 \
```

Comment:
Hm, Icinga Web creates tables with `utf8` explicitly: https://github.com/Icinga/icingaweb2/blob/master/etc/schema/mysql.schema.sql#L13

So `utf8` is fine here for now, but we should create an issue for Web.

Reply:
I suspect that they have not thought this through and will have to switch to utf8mb4 eventually, else you cannot use the full breath of utf8 in the relevant identifiers (usernames, groups, monitoring results).

Reply:
I mean, why wouldn't you want to name a group of windows servers 'those 💩current eater'